samples/Sentry.Samples.ME.Logging/Program.cs

```diff
@@ -26,7 +26,7 @@ private static void Main()
                 o.MinimumEventLevel = LogLevel.Error; // This level or above will result in event sent to Sentry
 
                 // Don't keep as a breadcrumb or send events for messages of level less than Critical with exception of type DivideByZeroException
-                o.AddLogEntryFilter((category, level, eventId, exception)
+                o.AddLogEntryFilter((_, level, _, exception)
```

Comment:
Might be good to keep these names here just since it's a sample program for informational purposes

Reply:
done